src/main/java/com/socrata/datasync/SMTPMailer.java

```diff
@@ -92,7 +92,9 @@ to true (the default), causes the transport to wait for the response to the QUIT
         final MimeMessage msg = new MimeMessage(session);
 
         // -- Set the FROM and TO fields --
-        msg.setFrom(new InternetAddress(userPrefs.getSmtpUsername()));
+        if (userPrefs.getSmtpUsername() != null && !userPrefs.getSmtpUsername().isEmpty()) {
+            msg.setFrom(new InternetAddress(userPrefs.getSmtpUsername()));
+        }
```

Comment:
This seems a little odd... without it, does the From field get populated automatically?